FEM/Output.h

```diff
@@ -247,6 +247,12 @@ class COutput : public GeoInfo, public ProcessInfo, public DistributionInfo
 	/// Tecplot share zone
 	bool tecplot_zone_share; // 10.2012. WW
 
+public:
```

Comment:
Be aware that you change accessibility of subsequent members too! E.g. `int mrank` becomes public too.

Solution: Remove this line and move control plane vector declaration to the existing `public:` area.